src/main/java/org/spongepowered/common/event/tracking/PhaseContext.java

```diff
@@ -98,6 +99,13 @@ public PhaseContext add(@Nullable NamedCause namedCause) {
         return this;
     }
 
+    public PhaseContext addCause(Cause cause) {
```

Comment:
Yes, added to the context, but don't deconstruct the cause to the context and then re-construct it.... it's wasteful.